packages/core/package.json

```diff
@@ -46,6 +46,7 @@
     "@silverhand/essentials": "^2.9.0",
     "@simplewebauthn/server": "^9.0.0",
     "@withtyped/client": "^0.7.22",
+    "bcrypt": "^5.1.1",
```

Comment:
can you use `hash-wasm`? it provides various hashing functions

Reply:
updated